gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/Table.java

```diff
@@ -322,14 +322,15 @@ public Builder toBuilder() {
   }
 
   @Override
-  public boolean equals(Object obj) {
-    return obj instanceof Table
+  public final boolean equals(Object obj) {
+    return this == obj
+        || obj instanceof Table
```

Comment:
> The method is final so even if extended that should not be a problem.

This PR removes the `final`, no?
I think that because it does not have final it would be better off with the getClass() check.